src/chart/line/LineSeries.ts

```diff
@@ -103,6 +103,8 @@ export interface LineSeriesOption extends SeriesOption<LineStateOption, ExtraSta
     showAllSymbol?: 'auto'
 
     data?: (LineDataValue | LineDataItemOption)[]
+
+    bolderWhenHover?: boolean
```

Comment:
Perhaps it can be `emphasis: { bolder?: boolean }`.
Which is similar to https://github.com/apache/incubator-echarts/blob/next/src/chart/scatter/ScatterSeries.ts#L49

Reply:
Sounds good! I will tweak it later.

Reply:
If users specify `emphasis.bolder` is `false` and `emphasis.lineStyle.width` to `5`, then should the line be bolder with a width of 5 pixel when hovering on it?

Reply:
I think we can just ignore `emphasis.bolder` if `emphasis.lineStyle.width` is specified.

Reply:
Looks good, it's just for emphasis state?

Reply:
@plainheart Perhaps `select` can also have that?

Reply:
@pissang I found it seems the line series has not implemented the logic about select state?

Reply:
I've tweaked the option `emphasis.bolder` to `emphasis.lineStyle.width: bolder` and I don't make changes about the `select` state since I find the line series doesn't support this state.

Reply:
Sure, I think it's good.